numpy/core/tests/test_regression.py

```diff
@@ -2225,6 +2225,13 @@ def test_reshape_size_overflow(self):
             new_shape = (2, 7, 7, 43826197)
         assert_raises(ValueError, a.reshape, new_shape)
 
+    def test_void_item_memview(self):
+        # gh-8129
+        va = np.zeros(300000, 'V4')
+        x = va[:1].item()
+        del va
+        x.tobytes()  # segfault?
```

Comment:
Always segfaults, or just now and then? Might move the trailing comment onto its own line above.